ner_v2/detectors/numeral/number/zh-TW/number_detection.py

```diff
@@ -149,4 +151,73 @@ def get_number_digit_by_digit(self, text=''):
 
     def get_number_with_digit_scaling(self, text=''):
```

Comment:
Add a description of what it does with the same example. 

2393576: sent in two batches [2 100 3 10 9], [ 3 1000 5 100 7 10]

Reply:
description updated